app/server.go

```diff
@@ -94,44 +77,30 @@ func main() {
 		}
 
 		proxy := httputil.NewSingleHostReverseProxy(remote)
-		httpsMux.HandleFunc("/patches/v1/connect/{conversation_id:[0-9]+}", reverseProxyHandler(proxy))
+		externalMux.HandleFunc(
+			"/patches/v1/connect/{conversation_id:[0-9]+}",
+			reverseProxyHandler(proxy),
+		)
 	}
 
-	httpsMux.PathPrefix("/").HandlerFunc(e.ReqHandler)
-	httpsMux.Use(logging, strictTransportSecurity)
-
-	httpsSrv := &http.Server{
-		Addr:         ":443",
-		ReadTimeout:  5 * time.Second,
-		WriteTimeout: 5 * time.Second,
-		IdleTimeout:  120 * time.Second,
-		Handler:      httpsMux,
-		TLSConfig: &tls.Config{
-			MinVersion:               tls.VersionTLS12,
-			CurvePreferences:         []tls.CurveID{tls.CurveP521, tls.CurveP384, tls.CurveP256},
-			PreferServerCipherSuites: true,
-			CipherSuites: []uint16{
-				tls.TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,
-				tls.TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA,
-				tls.TLS_RSA_WITH_AES_256_GCM_SHA384,
-				tls.TLS_RSA_WITH_AES_256_CBC_SHA,
-			},
-		},
-		TLSNextProto: map[string]func(*http.Server, *tls.Conn, http.Handler){},
-	}
+	externalMux.PathPrefix("/").HandlerFunc(e.ReqHandler)
+	externalMux.Use(logging)
 
-	e.RC.Transport = &http.Transport{TLSClientConfig: httpsSrv.TLSConfig}
+	externalSrv := makeServerFromMux(externalMux)
+	externalSrv.Addr = ":80"
 
-	// Start HTTPS server
 	go func() {
-		log.Fatal(httpsSrv.ListenAndServeTLS(certPath, privateKeyPath))
+		log.Fatal(externalSrv.ListenAndServe())
 	}()
 
-	// Create and start internal HTTP server
-	httpMux := mux.NewRouter()
-	httpMux.HandleFunc("/heimdall/v1/token/auth", e.PostTokenAuthHandler).Methods("POST")
-	httpMux.Use(logging)
-	httpSrv := makeServerFromMux(httpMux)
-	httpSrv.Addr = ":80"
-	log.Fatal(httpSrv.ListenAndServe())
+	// Create server for handling internal requests
+	internalMux := mux.NewRouter()
+	internalMux.HandleFunc(
+		"/heimdall/v1/token/auth",
+		e.PostTokenAuthHandler,
+	).Methods("POST")
+
+	internalSrv := makeServerFromMux(internalMux)
+	internalSrv.Addr = ":88"
```

Comment:
I think that we should use a port number over 1023 for this, since ports 0-1023 are the "well-known" ports.

Reply:
Does 8080 sound good?